public/app/features/alerting/unified/plugins/PluginOriginBadge.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useAsync } from 'react-use';
+
+import { Badge, Tooltip } from '@grafana/ui';
+
+import { getPluginSettings } from '../../../plugins/pluginSettings';
+
+interface PluginOriginBadgeProps {
+  pluginId: string;
+}
+
+export function PluginOriginBadge({ pluginId }: PluginOriginBadgeProps) {
+  const { value: pluginMeta } = useAsync(() => getPluginSettings(pluginId));
+
+  const logo = pluginMeta?.info.logos?.small;
+
+  const badgeIcon = logo ? (
+    <img src={logo} alt={pluginMeta?.name} style={{ width: '20px', height: '20px', borderRadius: '50%' }} />
```

Comment:
Should we be putting a border radius on the logo? Feels like we might end up obscuring it

Reply:
Haha, good catch. No idea why it's there 😄